sirepo/uri_router.py

```diff
@@ -204,6 +209,13 @@ def uri_for_api(api_name, params=None, external=True):
 
 
 class _Route(PKDict):
+    """ Holds all route information for an API.
+
+    Keys:
+        func (function): The function object for the API
+        cls (class): The class in the API's module that contains the API function.
+        func_name (str): The function's name (ex: 'api_admJobs').
```

Comment:
Great. Please add more doc for decl_uri, etc. Look in _init_uris to see all the fields.

Reply:
8f39fbdf6e35e7cd3105d1e0fbf9da2e60f893c7